backends/platform/android/org/scummvm/scummvm/ScummVMActivity.java

```diff
@@ -839,6 +838,18 @@ protected String getScummVMLogPath() {
 			} else return "";
 		}
 
+		@Override
+		protected void setCurrentGame(String target) {
+			Uri data = null;
+			if (target != null) {
+				data = Uri.fromParts("scummvm", target, null);
+			}
+			Intent intent = new Intent(Intent.ACTION_MAIN, data,
+				ScummVMActivity.this, ScummVMActivity.class);
+//			Log.d(ScummVM.LOG_TAG, "Setting new Intent: " + data);
```

Comment:
I would keep this Log output

Reply:
Micro comment. Else LGTM

Reply:
I changed the log message and commented it out although I am not really sure it's useful.